### ISSUE TITLE
Multiple Instances of Zigbee2MQTT in Home Assistant Supervised

### ISSUE THREAD

**Issue by AlohaFlyer** [zigbee2mqtt/hassio-zigbee2mqtt#272]
**Description of problem:**
Is your feature request related to a problem? Please describe
I am only able to cleanly install a single Zigbee2MQTT coordinator in my Home Assistant instance. I have a large system (189 lights) and need to use 4 coordinators on different Zigbee channels.

**Describe the solution you'd like**
I would like to see if we can build the Zigbee2MQTT HA Add On to allow for multiple coordinators.

**Describe alternatives you've considered**
Example: SmartThings and Hubitat can do this already. At the moment I am stuck using 4 SmartThings hubs as coordinators.

The alternative is to run Zigbee2MQTT for a single coordinator, then ZHA for another and something else for the other two.

It would be cleaner to support to run them all as the same system if possible.

I have tested many times, running a single large Zigbee mesh with good coordinators. But its never reliable until I split the system into smaller segments on different channels.

**Additional context**
Zigbee coordinators can really only handle 45-50 zigbee devices (In my case LEDVANCE lights) in a single mesh. Even with good routers it is not stable after about 50 devices. I want to run almost 200 lights hence 4 coordinators.

**Comment by AlohaFlyer** [comment]
I know. I would just like to run them cleanly in the same container.  I can do that with Hubitat hubs, or SmartThings hubs... wish I could do that with Z2M

**Comment by WhiteNoseThunderPants** [comment]
I don't see it likely in the same container, but you can run multiple Z2M containers from HA. Apparently, you can add the repository a second time by adding a / at the end of the URL. It then gives you a second instance of the Z2M add-on to install.

You have to make sure to change anything in the config that would conflict such as the port and data directory, etc.

**Comment by Gelloxx** [comment]
After reading @WhiteNoseThunderPants last comment I've played around and tried to get it working in my supervised HA instance. I was not able to add the repository a second time, but I tried a workaround that worked

 I've installed the Zigbee2mqtt Edge add-on in addition to the regular Zigbee2mqtt already installed, copied the original settings into the edge version, and changed the following:

In the Edge add-on setup:
- different data_path
- different mqtt base_topic
- changed serial port from 8888 to 8889
- changed the Socat tcp-listen port from 8485 to 8486
- setup a different channel from the original zigbee2mqtt instance

In the second coordinator (Sonoff Zigbee Bridge with tasmota):
- MQTT Topic same as new Edge base_topic
- changed port to 8889 in command line: backlog rule1 on system#boot do TCPStart 8889 endon ; rule1 1 ; template {"NAME":"Sonoff ZHABridge","GPIO":[56,208,0,209,59,58,0,0,0,0,0,0,17],"FLAG":0,"BASE":18} ; module 0

I was then able to start adding devices in the Edge instance while the first instance was kept running. So far, the two networks are working flawlessly in parallel.
